source/cursorManager.py

```diff
@@ -297,6 +297,9 @@ def _selectionMovementScriptHelper(self,unit=None,direction=None,toPosition=None
 				# 1. Caret at 1; selection (1, 1)
 				# 2. Shift+rightArrow: selection (1, 2)
 				newInfo.setEndPoint(oldInfo, "startToStart")
+		# _isSelectionAnchoredAtStart is not the same as _lastSelectionMovedStart,
+		# i.e. their default values are opposite and _isSelectionAnchoredAtStart is only updated if the selection has changed.
+		self._isSelectionAnchoredAtStart = not self._lastSelectionMovedStart

```

Comment:
On second thought there is a difference. But maybe it is actually the desired behavior.
Flipping around `_lastSelectionMovedStart` to `_isTextSelectionAnchoredAtStart` assumes that the end of the seletion is moving unless proven otherwise. This means that if the anchor position is unknown, braille will default to showing the end of the selection.
I'll give it a try and push an update when I'm satisfied.